src/components/Creations/Creations.tsx

```diff
@@ -99,6 +106,11 @@ const Creations = (props: Props) => {
           <div className={styles.sidebar}>
             <CategoryFilter i18n={{ title: t('categories_menu.title') }} items={categories} value={category} onClick={handleSetCategory} />
             <RarityFilter rarities={rarities} onChange={handleSetRarity} />
+            <SmartWearableFilter
```

Comment:
The `SmartWearableFilter` should be shown only when a wearable is being shown, would you mind adding a condition to show it under that circumstance? We have a function called `getCategoryName` that returns either `wearables` or `emotes` depending on the category.

Reply:
Ohh good catch, sure, added!